### PR TITLE
add `tqdm` stub

### PR DESCRIPTION
**Pull request by casperdcl**
failing tests due to `mypy` bug https://github.com/python/mypy/issues/8625

### CODE REVIEW

File: stubs/tqdm/METADATA.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+version = "4.62"
+python2 = true
```
Comment (srittau) [inline stubs/tqdm/METADATA.toml:2]: While tqdm supports Python 2, the stubs don't seem to be take Python 2 into account. In this case, just leave out the `python2` marker. Python 2 support in typeshed is not a priority anymore and hinders other improvements.

File: stubs/tqdm/tqdm/std.pyi [hunk 2]
```diff
@@ -0,0 +1,127 @@
+from _typeshed import Self
+from typing import Any, Dict, Iterable, Iterator, TypeVar
```
Comment (srittau) [inline stubs/tqdm/tqdm/std.pyi:2]: `Iterable` and `Iterator` should be imported from `collections.abc`.

File: stubs/tqdm/tqdm/std.pyi [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+from .utils import Comparable
+
+_T = TypeVar("_T")
+
+class tqdm(Comparable):
```
Comment (srittau) [inline stubs/tqdm/tqdm/std.pyi:8]: `tqdm` needs to be generic over `_T` for the type vars below to work.

**Review by srittau — COMMENTED** [review]
A few quick notes, not a thorough review yet.

**Comment by srittau** [comment]
mypy seems ok now, but this still fails, partly due to the issues I pointed out above.

**Comment by JelleZijlstra** [comment]
I was going to try to apply some fixes, but looks like the OP didn't give maintainers permission to push to their branch. Since they haven't responded in ~5 months, I'm going to close this PR. Feel free to reopen it and address Sebastian's comments whenever you're ready.

**Pull request closed without merging**